docs/USAGE-GUIDE.md

```diff
@@ -1032,6 +1044,18 @@ is supported by a given cipher preferences. It returns
 - -1 on any other errors
 
 
+### s2n\_connection\_set\_security\_policy
+
+```c
+int s2n_connection_set_versioned_security_policy(struct s2n_connection *conn, const char *version);
+```
+
+**s2n_connection_set_versioned_security_policy** sets the security policy override for the
+s2n_connection. Calling this function is not necessary unless you want to set the
+security policy on the connection to something different than what is in the s2n_config.
+This function is a wrapper for **s2n_connection_set_cipher_preferences**.
+
+
 ### s2n\_connection\_set\_cipher\_preferences
```

Comment:
We should update the documentation for `s2n_connection_set_cipher_preferences` to indicate that it is `S2N_DEPRECATED` API.

Reply:
Updated